DependencyInjection/FOSRestExtension.php

```diff
@@ -37,6 +40,29 @@ public function prepend(ContainerBuilder $container)
         if ($config['view']['view_response_listener']['enabled']) {
             $container->prependExtensionConfig('sensio_framework_extra', ['view' => ['annotations' => false]]);
         }
+
+        $bundles = $container->getParameter('kernel.bundles');
+        if (isset($bundles['WebProfilerBundle'])) {
+            $profilerConfigs = $container->getExtensionConfig('web_profiler');
+            $profilerConfig = $this->processConfiguration(new ProfilerConfiguration(), $profilerConfigs);
+
+            if ($profilerConfig['toolbar'] || $profilerConfig['intercept_redirects']) {
+                $path = '_profiler';
+                if ($profilerConfig['toolbar']) {
+                    $path .= '|_wdt';
+                }
+
+                $this->formatListenerDefaultRules[] = [
```

Comment:
hmm I actually wonder how this even worked before .. the order of rules matters .. so imho just appending the rule to the end is problematic as if someone as a ``^/`` rule defined, then this automatic rule will never be applied. so I guess we need to add this rule to the beginning of the array.